docs/solutions/sql_server_always_on_availability_groups/env-vars

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# OCI Account authentication information
+export TF_VAR_tenancy_ocid=<Tenancy OCID>
+export TF_VAR_user_ocid=<OCI User OCID that will deploy SQL AO>
+export TF_VAR_region=<Region to deploy SQL AO>
+export TF_VAR_private_key_path=<Private portion of API key associated with user>
+export TF_VAR_fingerprint=<Fingerprint of public portion of API key>
+export TF_VAR_private_key_password=<Private key password, if set, otherwise "">
+
+# Additional information required by TF template
+export TF_VAR_home_region=<Home Region - may be different than above>
+export TF_VAR_ssh_public_key=<public portion of SSH key used to log into instance>
+export TF_VAR_image_id=<Windows image OCID to use for SQL deployment>
```

Comment:
I would actually leave this out now that image_id is a map. Users will be less likely to have specify it, but specifying it is a little more annoying, since you now have to specify a map for this value.